js/base-ui.js

```diff
@@ -88,12 +88,15 @@ function baseUi () {
 
 		const handleClick = (clazz, evt) => $wrpBtnsMain.find(`.${clazz}`).trigger("click", evt);
 		$(`<li title="Map" class="choosemap"><span class="pictos" style="padding: 0 3px;">@</span></li>`).appendTo($ulBtns).click((evt) => handleClick(`choosemap`, evt));
-		$(`<li title="Background" class="choosebackground"><span class="pictos">a</span></li>`).appendTo($ulBtns).click((evt) => handleClick(`choosebackground`, evt));
+		if (d20plus.cfg.getOrDefault("canvas", "showBackground"))
+			$(`<li title="Background" class="choosebackground"><span class="pictos">a</span></li>`).appendTo($ulBtns).click((evt) => handleClick(`choosebackground`, evt));
```

Comment:
please use:

```
if (condition) statement
```

or 

```
if (condition) {
  statement
}
```

but _not_

```
if (condition)
  statement
```

as this is easy to trip over if one ever wants to add a second+ conditionally-run statement